spec/features/service_providers_spec.rb

```diff
@@ -117,3 +117,40 @@
     expect(page).to have_content('Approved')
   end
 end
+
+feature 'Users can access sps in their user group' do
+  context 'user is not the creator of the app' do
+    scenario 'service providers from a user group show on index' do
+      group = create(:user_group)
+      user1 = create(:user, user_group: group)
+      user2 = create(:user)
+      app = create(:service_provider, user_group: group, user: user2)
+      login_as(user1)
+
+      visit service_providers_path
+
+      expect(page).to have_content(app.friendly_name)
+    end
+
+    scenario 'user can edit a service provider from their user group', :js do
```

Comment:
should we add a scenario that a user can edit a service provider that they create?

why is js on here? better to turn off if we can for speed reasons ⏩